### DISCUSSION TITLE
Same Y scale for different tracks?

### DISCUSSION THREAD

**Discussion by albertoriva** [GMOD/jbrowse-components#2943] [Q&A]
Is it possible to tell JBrowse that two (or more) bigwig tracks should use the same Y scale? This is very useful when comparing the height of ChIP-seq or ATAC-seq peaks.

Thank you!

**Comment by cmdcolin** [comment]
this would indeed be a good feature but we don't currently have this https://github.com/GMOD/jbrowse-components/issues/1809

you can manually set the Y scale to be the same for each track using the track menu in the current code but making it automatic would be great

**Comment by albertoriva** [comment]
Thank you for your reply! Yes, it would be great to have this feature and
it's pretty much essential for me: I have pipelines that generate track
hubs automatically (sometimes with tens of tracks) and I wouldn't be able
to set the scale manually for all of them - also because the appropriate
scale changes depending on what region is being displayed. As far as I
know, the WashU browser is the only one that has this feature.

I'll follow the other discussion, thank you!

Alberto

**Reply by cmdcolin** [reply]
hello @albertoriva some development on a "multibigwig" track has started here. if you have any interest in testing it out and have any questions or feedback, the current branch is here https://github.com/GMOD/jbrowse-components/pull/3043

it has the ability to "add multiple bigwig tracks to a selection" in the track selector, which creates a shopping cart, and then you can create a multibigwig from items in the shopping cart
